src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/shaded/org/yaml/snakeyaml/nodes/Tag.java

```diff
@@ -61,8 +64,13 @@
         //
         Set<Class<?>> timestampSet = new HashSet<Class<?>>();
         timestampSet.add(Date.class);
-        timestampSet.add(java.sql.Date.class);
-        timestampSet.add(Timestamp.class);
+        try {
+            timestampSet.add(ClassLoader.getSystemClassLoader().loadClass("java.sql.Date"));
+            timestampSet.add(ClassLoader.getSystemClassLoader().loadClass("java.sql.Timestamp"));
+        } catch (ClassNotFoundException e) {
+            LOG.log(Level.INFO, "java.sql.Date and / or java.sql.Timestamp are not available. " +
+                  "You are probably on Java 11, if so, it's ok.", e);
+        }
```

Comment:
Upside: this looks like you got the right fix :-) https://bitbucket.org/asomov/snakeyaml/src/d318aef91b9df07b369eba6c553a8457b05e0726/src/main/java/org/yaml/snakeyaml/nodes/Tag.java?at=default&fileviewer=file-view-default#Tag.java-64:70

Downside: I actually think the right fix would be to upgrade snakeyaml. But seems like this is not yet released. So this looks fine.

I commented in https://bitbucket.org/asomov/snakeyaml/issues/422/provide-a-stripped-down-library-for-just to ask for a release.

I think in the meantime this fix looks correct.

Reply:
Upgrading snakeyaml should be done in a dedicated PR imo, even if it was released with the fix.

Reply:
Yeah, but then upgrading snakeyaml **would be** the fix. But as it's not released, that's fine.

Reply:
Also, got feedback from the snakeyaml maintainer in the issue linked above: release with this fix is reflected in February.